javagen/src/main/java/com/azure/autorest/util/ReturnTypeDescriptionAssembler.java

```diff
@@ -135,12 +139,13 @@ private boolean isGenericTypeClassSubclassOf(IType type, Class<?>... parentClass
 
     private Class<?> getGenericClass(GenericType type) {
         String className = String.format("%s.%s", type.getPackage(), type.getName());
-        try {
-            return Class.forName(className);
-        } catch (ClassNotFoundException e) {
-            logger.warn(String.format("class %s not found!", className), e);
-            return null;
-        }
+        return TYPE_CLASS_MAP.computeIfAbsent(className, key -> {
+            try {
+                return Optional.of(Class.forName(key));
+            } catch (ClassNotFoundException e) {
+                logger.warn(String.format("class %s not found!", key), e);
+                return Optional.empty();
+            }
+        }).orElse(null);
     }
```

Comment:
A cache here.

We need future work on this class, as generic-core class e.g. `Response` not be able to be loaded from runtime.

Reply:
I wonder why. Does this mean codegen won't depend on generic-core?

Reply:
I don't know.

Strickly speaking, codegen does not need to depends on azure-core either. We just have it here for convenience, not necessity.